### ISSUE TITLE
Installation?

### ISSUE THREAD

**Issue by comod** [Skype4Py/Skype4Py#66]
Hi!

Why you dont provide a Installation Guide?

I Think i have all steps done, but it dont work. I Always get a black window without any output... It looks like skype.Attach() will end in a endless loop or something similar...

**Comment by davsebamse** [comment]
Can you please provide what you have done, python version and operating system

**Comment by comod** [comment]
I have installed setuptools, skype4Py by typing "python setup.py install", i also have installed powershell, python 2.7 (first 3.0), add PATH Variable, all things like in this video https://www.youtube.com/watch?v=6p-1cBvDifA ... i am using Win7, but at least i want a Linux Solution.

**Comment by davsebamse** [comment]
It might be that the newest versions of skype do not support Skype4Py. I will look into it soon-ish.

Just to summarize. You can import the module but it is in the skype.attach that it hangs?

**Comment by comod** [comment]
Yes! Where can i find a older Version for Skype without downloading a Virus ;-) ?

**Comment by davsebamse** [comment]
If possible, try on linux. I know that is an older version

**Comment by comod** [comment]
It Works on Linux ! :-) ... Here is my Installation Guide:

Download Ubuntu: http://www.heise.de/download/264e2de9fdfd96e49a232ba76e279cd8-1463411395-22191449.html
Create VMware with it

> sudo add-apt-repository "deb http://archive.canonical.com/ubuntu $(lsb_release -sc) partner" 
> sudo apt-get update
> sudo apt-get install skype
> skype
> sudo apt-get install git
> cd /home/myUser/
> git clone https://github.com/Skype4Py/Skype4Py.git
> sudo apt install python-pip
> pip install -U setuptools
> cd Skype4Py
> sudo python setup.py install
> sudo apt-get install python-dbus
> sudo apt-get virtualenv
> virtualenv --system-site-packages dbus-venv
> sudo apt-get install python-gobject

Create a file test.py:

import Skype4Py

# Create an instance of the Skype class.
skype = Skype4Py.Skype()

# Connect the Skype object to the Skype client.
skype.Attach()

# Obtain some information from the client and print it out.
print 'Your full name:', skype.CurrentUser.FullName
print 'Your contacts:'
for user in skype.Friends:
    print '    ', user.FullName

> chmod 775 test.py
> python test.py

**Comment by miohtama** [comment]
For installing Python packages please never do `sudo pip install` as this is a broken practice.

Here is the official Python package installation guide:

https://packaging.python.org/en/latest/installing/
